ext/iperf3/iperf_client_api.c

```diff
@@ -104,9 +104,10 @@ iperf_create_streams(struct iperf_test *test, int sender)
 #endif /* HAVE_TCP_CONGESTION */
 
 	if (sender)
-	    FD_SET(s, &test->write_set);
+		FD_SET(s, &test->write_set);
 	else
-	    FD_SET(s, &test->read_set);
+		FD_SET(s, &test->read_set);
+	FD_SET(s, &test->err_set);
```

Comment:
If we cannot use curly braces in above if-statement, maybe even an empty line before this line to have a separation to the if-else.